code/modules/antagonists/macho_man/abilities/_macho_man_mob.dm

```diff
@@ -416,6 +416,10 @@ var/list/snd_macho_idle = list('sound/voice/macho/macho_alert16.ogg', 'sound/voi
 	icon_state = "machopants"
 	item_state = "machopants"
 
+	New()
+		. = ..()
+		AddComponent(/datum/component/toggle_underwear)
+
```

Comment:
```suggestion
```
This only covers the bottom half and so shouldn't hide underwear. Also wrestlers totally wear pants under their outfits.